### DISCUSSION TITLE
V Rising server request

### DISCUSSION THREAD

**Discussion by dino1816** [GameServerManagers/LinuxGSM#4576] [General]
First time doing this, not sure if this is the right channel to do it but here I am, can V Rising servers be somehow added to the LGSM website? Did they release linux server files for that?

Also how about DayZ standalone (stable branch) still no files?

**Comment by q220** [comment]
There are no server binaries for Linux I noticed, only way right now seems to use mono/wine?

**Reply by h3o66** [reply]
it is in general, but not supported by lgsm.

steamdb app for reference:
https://steamdb.info/app/1829350/
